src/pages/SignUp/SignUp.scss

```
@@ -0,0 +1,169 @@
+.signUp {
+  position: relative;
+  min-width: 1000px;
+}
+
+.signUpHeader {
+  margin: 50px;
+
+  .signUpLogo {
+    width: 150px;
+    margin-left: 150px;
+    padding-top: 7px;
+  }
+  .signUpTitle {
+    font-family: 'KimjungchulGothic-Bold';
+    font-size: 35px;
+    border-bottom: 1px solid black;
+    margin-left: 225px;
+    margin-right: 225px;
+    padding-top: 55px;
+  }
+  .signUpLine {
+    margin: 0 150px 0 150px;
+  }
```

```suggestion
  .signUpTitle {
    font-family: 'KimjungchulGothic-Bold';
    font-size: 35px;
    border-bottom: 1px solid black;
    margin: 0 225px;
    padding-top: 55px;
  }
  .signUpLine {
    margin: 0 150px;
  }
```
축약할 수 있는 부분은 축약해주세요!